docs/en/getting-started/get-started-stack.asciidoc

```diff
@@ -109,9 +109,30 @@ ifeval::["{release-state}"!="unreleased"]
 
 ["source","sh",subs="attributes,callouts"]
 ----------------------------------------------------------------------
-curl -L -O https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-{elasticsearch_version}.tar.gz
-tar -xzvf elasticsearch-{elasticsearch_version}.tar.gz
-cd elasticsearch-{elasticsearch_version}
+curl -L -O https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-{elasticsearch_version}-darwin-x86_64.tar.gz
+tar -xzvf elasticsearch-{elasticsearch_version}-darwin-x86_64.tar.gz
+cd elasticsearch-{elasticsearch_version}-darwin-x86_64
+./bin/elasticsearch
+----------------------------------------------------------------------
+
+endif::[]
+
+
+[[linux]]*linux:*
+
+ifeval::["{release-state}"=="unreleased"]
+
+Version {version} of {es} has not yet been released.
+
+endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
+
+["source","sh",subs="attributes,callouts"]
+----------------------------------------------------------------------
+curl -L -O https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-{elasticsearch_version}-linux-x86_64.tar.gz
+tar -xzvf elasticsearch-{elasticsearch_version}-linux-x86_64.tar.gz
+cd elasticsearch-{elasticsearch_version}-linux-x86_64
```

Comment:
The extracted directory name hasn't changed, so this should still be `elasticsearch-{elasticsearch_version}`

Reply:
Thanks @rjernst I implemented those changes